pages/analysis.md

```diff
@@ -1,15 +1,36 @@
+
+This image overlays major world events onto our data to see if they have effected migrations to the United States. The events are the California Gold Rush, the Irish Potato Famine, the U.S. Civil War, the Unification of Germany, the Angell Treaty of 1880, World War I,The Great Depression, World War II, the Guatamelan Civil War, the Honduras Wars & Corruption, and the El Salvadorean Civil War.
+
+---
+
+We are having problems with switching the axis in excel, so we were trying to find ways to flip it and make the graph easier on us to upload on tableau. Because the way tableau is with grouping we are having problems with it if the graph was not flipped making the year switched to the side. With the help of the group and Professor Thomas we were able to flip the information in Excel. Professor Thomas helped by showing me how to flip the information in excel by just copying the Data and selecting paste special then press transpose. That is how Professor Thomas helped teach me this method that helped solve this major problem that my group had that would mess up our visualizations. My Group members helped by searching a method before Professor Thomas helped us out with Excel. We also did not find any more problems that we encountered throughout the project so the data in excel would be the only problem everything else was not that hard or bad to get through. We found out that some events did have an effect on immigration in the United States. From Laws, and Major events in world history starting from 1825. A Major event that caused a huge drop of immigration to the United States is World War II. Are graph showed that the U.S. were strict about Immigrations during Both World War I and World War II. With both wars it showed a huge decline in immigration mostly heading towards World War II. I would think the reason for the decline of migration to the United States would be because of National Security since they are at war with many countries from Europe and Asia during WWII. In a Visualization that I made about major events and wars that would cause big changes in immigrations, there was also a major growth in migration to the United Sates by Europeans during 1850’s and a cause would be the Irish Potato famine. Another major event that happened that caused immigration to spike would be Angell Treaty of 1880 that caused the Asian migrants spike really high. Another Visualization I made was about laws that were passed that would have had an effect towards immigration. For example, there was a decline of immigration after the 1917 Immigration Act was passed which showed that during that time there was a strict immigration policy in the United States.
```

Comment:
You are no longer having problems with transposing the axes (that is the plural of axis) in Excel, so this text should be removed. You already mentioned that you had to do that in the workflow section.

Also, break up huge paragraphs like this. Especially when you are talking about a process like this, it makes it harder to read (I'm not joking, there are studies, and it matters especially for readers on the web).

Instead, you should bring the paragraphs that talk about the visualizations above from the workflow section, since those paragraphs do much more analysis than the text above.

In fact, a significant part of this paragraph is still talking about workflow and should also be a part of the workflow section. So, you should switch some of the things on this page to workflow (the parts that talk about the process) to that page and some parts of that page (the parts that talk about the actual results of the visualizations) to this page.

That will make everything much easier to understand. It feels like you kind of blended the two pages.

This page should be reserved for explaining each of the visuals and then making conclusions about the results!